examples/app.py

```diff
@@ -157,29 +168,39 @@
 InvenioRecordsUI(app)
 
 
-@app.cli.group()
-def fixtures():
-    """Command for working with test data."""
+app.register_blueprint(accounts_blueprint)
 
 
-def create_object(bucket, file_name, stream):
+def create_object(bucket, file_name, stream, checksum):
     """Object creation inside the bucket using the file and its content."""
     obj = ObjectVersion.create(bucket, file_name, stream=stream)
     rec_uuid = uuid4()
-    provider = RecordIdProvider.create(object_type='rec', object_uuid=rec_uuid)
+
+    provider = (RecordIdProvider.create(object_type='rec',
+                object_uuid=rec_uuid))
+
     data = {
         'pid_value': provider.pid.pid_value,
         'files': [
```

Comment:
When using invenio-records-files this should probably be changed to ``files`` -> ``_files``. See http://pythonhosted.org/invenio-records-files/usage.html, especially http://pythonhosted.org/invenio-records-files/usage.html#dumping-files